### ISSUE TITLE
Plugins: show warning to user if some resources were not removed during removal of plugin

### ISSUE THREAD

**Issue by dcdenu4** [natcap/invest#1666]
I'm seeing:
```
 WARNING conda.gateways.disk.delete:unlink_or_rename_to_trash(182): Could not remove or rename C:\Users\ddenu\miniforge3\envs\invest_plugin_schistosomiasis\Library\share\proj\proj.db.  Please remove this file manually (you may need to reboot to free file handles)
[1] WARNING conda.gateways.disk.delete:unlink_or_rename_to_trash(182): Could not remove or rename C:\Users\ddenu\miniforge3\envs\invest_plugin_schistosomiasis\Scripts\invest.exe.  Please remove this file manually (you may need to reboot to free file handles)
[1] [11:24:47.964] [undefined] successfully removed plugin
```

**Comment by dcdenu4** [comment]
It could be because I had the environment activated in another shell.

**Comment by dcdenu4** [comment]
Actually, what was happening was I had a few zombie InVEST processes running that were never properly ended. These must have been using the plugin env, because once I killed these I was able to force remove the plugin env.

**Comment by emlys** [comment]
I don't think we can guarantee that no processes outside the workbench are using the env, but we can at least show this warning to the user

**Comment by emlys** [comment]
@dcdenu4 I could not reproduce this on mac - I tried launching the workbench, opening another shell and running `invest serve` in the plugin environment, and then removing the plugin via the workbench interface. The removal succeeded and didn't show any warnings. The plugin environment directory remained, but all packages were removed from it. 

Maybe this specific failure mode is unlikely enough in regular use that we don't need to handle it?
